SparkPost/support-docs
# WE-310 Decrease build times

> Build time: 7m 9s. Total deploy time: 11m 59s

**What Changed**
- Removes nav data sourcing from `getStaticProps`
- Adds `yaml-loader` and loads nav data via webpack instead of on each route

**How to verify**
- Run the site locally or visit the deploy preview
- Verify the navigation works correctly
- Verify build times are lowered in Netlify
